README.md

```diff
@@ -0,0 +1,51 @@
+## README
+
+# Setup
+
+* Install Rust
+    * Note: Latest [Rust version](https://www.rust-lang.org)
+    ```
+    curl https://sh.rustup.rs -sSf | sh
+    ```
+    * Restart the terminal shell session
+        ```
+        source ~/.bash_profile
+        ```
+    * Check if `rustc` command is found
+        ```
+        rustc --version
+        ```
+
+* Build
+    * Note: ./Cargo.toml configuration file builds source code in ./server/src and ./cli/src
```

Comment:
Not entirely sure we need this, cause it's a pretty regular setup and that's how cargo works.